app/settings.go

```diff
@@ -14,7 +14,7 @@ import (
 // SettingsSchema is used for loading and storing global settings
 type SettingsSchema struct {
 	// these items are used for global settings load
-	ThemeName    string  `json:"theme"`
+	ThemeVariant string  `json:"theme"`
```

Comment:
This is part of our public API and cannot be changed outside a deprecation cycle and breaking release

Reply:
Yes, I forgot it. Should we keep `ThemeName` instead of `ThemeVariant`? I renamed it, because I think the new name is more clear based on the usage

Reply:
To get this in a bugfix we have to keep it as-is. We could look to change it in the future

Reply:
Got it, will update :)

Reply:
done